tutorials/combining-feature-collections/index.md

```diff
@@ -47,10 +47,10 @@ var moreNewFeatures = ee.FeatureCollection.randomPoints(utahGeometry, 25, 1);
 
 ## Combine the `ee.FeatureCollection` objects
 
-Next, create a new `ee.FeatureCollection` using a list of the desired `ee.FeatureCollection`s to combine. Then, flatten them into a single `ee.FeatureCollection`.
+Next, create a new `ee.FeatureCollection` by merging `newFeatures` and `moreNewFeatures`.
 
 ```js
-var combinedFeatureCollection = ee.FeatureCollection([newFeatures, moreNewFeatures]).flatten();
```

Comment:
@n-clinton Is this what you had in mind?

Reply:
LGTM